saleor/dashboard/templatetags/utils.py

```diff
@@ -27,9 +27,15 @@ def construct_get_query(context, **params):
 
 @register.filter
 def is_versatile_image_ppoi_click_widget(field):
+    print(type(field.field.widget), isinstance(field.field.widget, VersatileImagePPOIClickWidget))
```

Comment:
Debug code left

Reply:
Could we add a docstring to this function explaining in what cases we expect the field to be an instance of `VersatileImagePPOIClickWidget `? This logic is a bit confusing and we ourselves had problems with figuring out in what cases which of the two image widgets is used.